pkg/cluster/spec/monitoring.go

```diff
@@ -60,6 +60,7 @@ type PrometheusSpec struct {
 	AdditionalScrapeConf  map[string]any         `yaml:"additional_scrape_conf,omitempty" validate:"additional_scrape_conf:ignore"`
 	ScrapeInterval        string                 `yaml:"scrape_interval,omitempty" validate:"scrape_interval:editable"`
 	ScrapeTimeout         string                 `yaml:"scrape_timeout,omitempty" validate:"scrape_timeout:editable"`
+	RunCommand            string                 `yaml:"run_command,omitempty" validate:"run_command:editable"`
```

Comment:
Use string[] for better handling of multiple commands

Reply:
Have been updated